dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/PushNowActionlet.java

```diff
@@ -14,122 +18,135 @@
 import com.dotmarketing.portlets.workflows.model.WorkflowProcessor;
 import com.dotmarketing.util.Logger;
 import com.dotmarketing.util.UtilMethods;
+import com.liferay.portal.model.User;
 
 import java.util.ArrayList;
 import java.util.Date;
 import java.util.List;
 import java.util.Map;
 
+/**
+ * This Workflow Actionlet allows users to Push Publish a piece of Content when executing a Workflow Action. It takes
+ * two parameters:
+ * <ol>
+ *     <li><b>Name of the Environment (String):</b> The name of the Push Publishing environment that will receive the
+ *     Contentlet.</li>
+ *     <li><b>Force the Push? true or false (String):</b> Type in true if the push will be forced. Otherwise, type in
+ *     false. Defaults to false.</li>
+ * </ol>
+ * It's worth noting that, as its name implies, this Workflow Actionlet <b>allows users to {@code Push} content ONLY,
+ * and NOT to {@code Remove} or {@code Push Remove}.</b>
+ *
+ * @author Oscar Arrieta
+ * @version 3.2
+ * @since Mar 4, 2015
+ */
 public class PushNowActionlet extends WorkFlowActionlet {
 
     private static final long serialVersionUID = 1L;
+
+    private static final String ENVIRONMENT_DELIMITER = ",";
+    private static final String ACTIONLET_NAME = "Push Now";
+    private static final String ACTIONLET_DESCRIPTION = "This actionlet will automatically publish the the content " +
+            "object to the specified environment(s). Multiple environments can be separated by a comma (',')";
+    private static final String PARAM_ENVIRONMENT = "environment";
+    private static final String PARAM_FORCE_PUSH = "force";
+
     private PublisherAPI publisherAPI = PublisherAPI.getInstance();
+    private EnvironmentAPI environmentAPI = APILocator.getEnvironmentAPI();
+    private BundleAPI bundleAPI = APILocator.getBundleAPI();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'bundleAPI' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5077051)